packages/bento-design-system/src/DateField/DateField.css.ts

```diff
@@ -0,0 +1,146 @@
+import { createVar, style } from "@vanilla-extract/css";
+import { bentoSprinkles } from "../internal";
+import { strictRecipe } from "../util/strictRecipe";
+
+export const dayRadius = createVar();
+
+export const calendar = [
+  bentoSprinkles({
+    background: "backgroundPrimary",
+    position: "absolute",
+    display: "flex",
+    outline: "none",
+    overflowY: "auto",
+  }),
+];
+
+export const dateFieldRecipe = strictRecipe({
+  variants: {
+    validation: {
+      valid: {},
+      invalid: {},
+      notSet: {},
+    },
+    isFocused: {
+      true: {},
+    },
+  },
+  compoundVariants: [
+    {
+      variants: {
+        validation: "valid",
+        isFocused: true,
+      },
+      style: bentoSprinkles({
+        boxShadow: { default: "outlineInputFocus", hover: "outlineInputFocus" },
+      }),
+    },
+    {
+      variants: {
+        validation: "invalid",
+        isFocused: true,
+      },
+      style: bentoSprinkles({
+        boxShadow: { default: "outlineNegativeStrong", hover: "outlineNegativeStrong" },
+      }),
+    },
+  ],
+});
+
+export const input = bentoSprinkles({
+  outline: "none",
+  background: "primaryTransparentEnabledBackground",
+  width: "full",
+});
+
+export const weekDay = [
+  style({ display: "inline-flex" }),
```

Comment:
mi spieghi meglio a cosa serve `inline-flex` in questo caso?

Reply:
a occhio a niente... penso derivi da esperimenti fatti prima di usare `Tiles`. `flex` sembra funzionare altrettanto bene